apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -59,7 +171,7 @@ handle_request('GetBlockLatest', Req, _Context) ->
 handle_request('GetBlockPending', Req, _Context) ->
     get_block(fun aehttp_logic:get_block_pending/0, Req, json, false);
 
-handle_request('GetKeyBlockByHeight', Req, _Context) ->
+handle_request('GetKeyBlockByHeightObsolete', Req, _Context) ->
```

Comment:
This is a duplicated endpoint. Will its removal be a subject to another PR? (I strongly suspect the path `/block/height/{height}` is being used by Python tests so its removal will require some changes there)

Reply:
This will be removed in separate PR.